services/transformer/features.go

```diff
@@ -38,12 +40,21 @@ var defaultTransformerFeatures = `{
 	"regulations": ["AM"],
   }`
 
-func NewFeaturesService(ctx context.Context, config FeaturesServiceConfig) FeaturesService {
+func NewFeaturesService(ctx context.Context, config *config.Config, featConfig FeaturesServiceConfig) FeaturesService {
 	handler := &featuresService{
 		features: json.RawMessage(defaultTransformerFeatures),
 		logger:   logger.NewLogger().Child("transformer-features"),
 		waitChan: make(chan struct{}),
-		config:   config,
+		options:  featConfig,
+		client: &http.Client{
+			Transport: &http.Transport{
+				DisableKeepAlives:   config.GetBool("Transformer.Client.disableKeepAlives", true),
```

Comment:
shouldn't this be false by default?
```suggestion
				DisableKeepAlives:   config.GetBool("Transformer.Client.disableKeepAlives", false),
```

Reply:
I see it to be true on transformer [here](https://github.com/rudderlabs/rudder-server/blob/master/processor/transformer/transformer.go#L221)

Reply:
Changed it to false anyways